app/src/main/java/com/android/feedme/ml/TextRecognition.kt

```diff
@@ -220,7 +219,7 @@ fun parseResponse(responseBody: String?, forIngredientFound: (IngredientMetaData
       val unitString = ingredientObject.optString("unit", "")
       val unit = getMeasureUnitFromString(unitString)
       forIngredientFound(
-          IngredientMetaData(quantity, unit, Ingredient(ingredient, "DEFAULT_TYPE", "DEFAULT_ID")))
+          IngredientMetaData(quantity, unit, Ingredient(ingredient, "NO_ID", false, false)))
```

Comment:
I saw that "`NO_ID"` was the default value at Ingredient creation, but here why : 

- don't we give a real idea to ingredient ?
- if we don't, why are we initialising it to `"NO_ID"` even though it's already the deafult value at creation ?

Reply:
This is a problem with how our model works, elements that are detected need to be transformed into `Ingredient` types before knowing or not  if their Ingredient exist in Firebase. So this allows me to distinguish between Ingredients that are actually in the database or not. I agree that this isn't optimal, but for now it works.

Reply:
It's just the way I code, we could remove it if necessary. I find it easier to code like this :)

Reply:
Ok no problem, it's datails anyway :)